frontend/src/components/common/Header.svelte

```diff
@@ -3,20 +3,15 @@
   import { page } from '$app/stores';
   import { to } from '$lib/routes';
   import { _ } from 'svelte-i18n';
-  import { locale } from 'svelte-i18n';
   import { Transition } from '@rgossiaux/svelte-headlessui';
   import user from '$stores/user';
   import { logOut } from '$utils/session';
-
   import * as Profile from './Header/Profile';
   import type { UserMenuItem } from './Header/Profile/Desktop.svelte';
   import AuthenticationButtons from './Header/AuthenticationButtons.svelte';
+  import Langswitcher from './Langswitcher.svelte';
```

Comment:
I would consider name it `LanguageSwitcher` because `Language` and `Switcher` are two different words

Reply:
> I would consider name it `LanguageSwitcher` because `Language` and `Switcher` are two different words

done